snapcraft.yaml

```diff
@@ -14,23 +14,48 @@ architectures:
   - build-on: arm64
   - build-on: armhf
 
+plugs:
+  sys-devices-virtual-info:
```

Comment:
there's a policy of naming this like the path being granted access to, might want to double check with security if this is the right name

Reply:
I've only found [this](https://snapcraft.io/docs/system-files-interface):
> Note, when declaring an instance of the system-files plug as above, it should be named with a descriptive name that indicates to a user what access it grants. In this case, the name etc-foo is used to reflect the access to /etc/foo.

I thought that name is descriptive enough. But if not, I can also change it to `sys-devices-virtual-dmi-id`.